fidesctl/src/fidesctl/core/api_helpers.py

```diff
@@ -75,23 +75,27 @@ def get_server_resource(
 
 def list_server_resources(
     url: str,
-    resource_type: str,
     headers: Dict[str, str],
+    resource_type: str,
+    exclude_keys: List[str],
 ) -> List[FidesModel]:
     """
     Get a list of resources from the server and return them as parsed objects.
 
     Returns an empty list if the given endpoint is valid but has no resources.
```

Comment:
according to this it returns an empty list only if the resource is valid, but it looks like the code was updated to always return an empty list?

Reply:
Yeahhh let me know what you think but I wanted to create a new issue on adding error handling to the api_helpers functions. Even though what the comment is saying is technically true, the logic is missing right now to actually do this properly. My proposal was to create an issue so that `get_server_resources` `get_server_resource` and `list_server_resources` raise errors when failures happen. The reason I didn't want to throw this in here is that I think there is some callers of these functions that would break if I added this. For instance in `relationships.py` we call `get_server_resources` without knowing the resource type so we need to make changes there to properly handle other return codes as we should.